src/host/screenInfo.cpp

```diff
@@ -1998,6 +2005,13 @@ void SCREEN_INFORMATION::UseMainScreenBuffer()
         s_RemoveScreenBuffer(psiAlt); // this will also delete the alt buffer

```

Comment:
I suspect the issue @lhecker found stems from the deletion of psiAlt here before we establish a reference to its cursor :)

Reply:
Yikes. Sorry guys. I don't know why I thought it was a good idea to add that code _after_ the line that clearly says it deletes the alt buffer. Will try and fix ASAP.

Funnily enough I can't actually reproduce the crash with `cmatrix` or `cacafire`, but maybe it only crashes in a debug build?